src/msg/compressor_registry.h

```diff
@@ -25,10 +25,10 @@ class CompressorRegistry : public md_config_obs_t {
   void handle_conf_change(const ConfigProxy& conf,
                           const std::set<std::string>& changed) override;
 
-  Compressor::CompressionAlgorithm pick_method(uint32_t peer_type,
+  TOPNSPC::Compressor::CompressionAlgorithm pick_method(uint32_t peer_type,
```

Comment:
why is that?

Reply:
to avoid function conflicts with seastar environment. [/include/common_fwd.h](https://github.com/ceph/ceph/blob/master/src/include/common_fwd.h)